src/test/Tests/Integration.hs

```diff
@@ -2,7 +2,7 @@ module Tests.Integration (integrationTests) where
 
 import Test.Tasty (TestTree, testGroup)
 
-import Common (testContract, testContractV, solcV, testContract', checkConstructorConditions, passed, solved, solvedLen, solvedWith, solvedWithout, gasInRange)
```

Comment:
Can we keep these tests? 😄

Reply:
I think these tests failed with the new hevm, but maybe you fixed them already?

Reply:
actually, I think I intended to remove the gas features since no one is using them (and they are also incomplete 😅 )